src/components/standalone/security/ips/IpsSuppressedAlerts.vue

```diff
@@ -0,0 +1,248 @@
+          <NeTableHeadCell column-key="direction" sortable @sort="onSort">
+            {{ t('standalone.ips.direction') }}
+          </NeTableHeadCell>
+          <NeTableHeadCell column-key="ip" sortable @sort="onSort">
+            {{ t('standalone.ips.address') }}
+          </NeTableHeadCell>
+          <NeTableHeadCell>
+            <!-- no header for actions -->
+          </NeTableHeadCell>
+        </NeTableHead>
+        <NeTableBody>
+          <NeTableRow v-if="filteredAlerts.length < 1">
+            <NeTableCell colspan="4">
+              <NeEmptyState
+                :description="t('common.try_changing_search_filters')"
+                :icon="faCircleInfo"
+                :title="t('standalone.ips.no_suppressed_alerts')"
+                class="bg-white dark:bg-gray-950"
+              >
+                <NeButton kind="tertiary" @click="filter = ''">
+                  {{ t('common.clear_filters') }}
+                </NeButton>
+              </NeEmptyState>
+            </NeTableCell>
+          </NeTableRow>
+          <NeTableRow v-else v-for="item in paginatedItems" :key="`${item.ip}-${item.direction}`">
+            <NeTableCell :data-label="t('standalone.ips.suppressed_alert_description')">
+              {{ item.description }}
+            </NeTableCell>
+            <NeTableCell :data-label="t('standalone.ips.suppressed_alert_id')">
+              {{ item.id }}
+            </NeTableCell>
+            <NeTableCell :data-label="t('standalone.ips.direction')">
+              <template v-if="item.direction == 'by_src'">
+                {{ t('standalone.ips.source') }}
+              </template>
+              <template v-else>
+                {{ t('standalone.ips.destination') }}
+              </template>
+            </NeTableCell>
+            <NeTableCell :data-label="t('standalone.ips.address')">
+              {{ item.ip }}
+            </NeTableCell>
+            <NeTableCell :data-label="t('common.actions')">
+              <div class="flex justify-end">
+                <NeDropdown :items="dropDownActions(item)" :align-to-right="true" />
+              </div>
+            </NeTableCell>
+          </NeTableRow>
+        </NeTableBody>
+        <template #paginator>
+          <NePaginator
+            :current-page="currentPage"
+            :nav-pagination-label="t('ne_table.pagination')"
+            :next-label="t('ne_table.go_to_next_page')"
+            :page-size="pageSize"
+            :page-size-label="t('ne_table.show')"
+            :previous-label="t('ne_table.go_to_previous_page')"
+            :range-of-total-label="t('ne_table.of')"
+            :total-rows="suppressedAlerts.length"
+            @selectPageSize="(size: number) => { pageSize = size }"
+            @select-page="(page: number) => { currentPage = page }"
+          />
+        </template>
+      </NeTable>
+    </template>
+    <NeEmptyState v-else :icon="faShield" :title="t('standalone.ips.no_suppressed_alerts')">
```

Comment:
Minor here we need a "No suppressed alerts" string (instead of "No suppressed alerts found")